cli/ops/cephadm_ansible.py

```diff
@@ -137,8 +137,14 @@ def exec_cephadm_preflight(node, build_type, ibm_build=False, repo=None):
     # Set ceph origin with RN or IBM
     extra_vars = {"ceph_origin": build_type}
 
+    s3_repo = False
+
+    # Check if the repo is an s3.upshift repo
+    if "s3.upshift.redhat.com" in repo:
+        s3_repo = True
+
     # Set custom repository
-    if repo:
+    if repo and not s3_repo:
```

Comment:
Won't below condition works ?
```
if repo and not "s3.upshift.redhat.com" in repo:
```

Reply:
Yes, this will also work, initially I had added different conditions with boolean values and later changed it.
Will make this change.

Reply:
Done